app/components/shared/navbar_component/navbar_component.html.slim

```diff
@@ -23,5 +23,8 @@ nav
         li.navbar-component__navigation-spacer aria-hidden="true" tab-index="-1"
         li.navbar-component__navigation-item--right class=active_link_class(new_jobseeker_session_path)
           = link_to t("buttons.sign_in"), new_jobseeker_session_path, class: "govuk-header__link"
-        li.navbar-component__navigation-item--right class=active_link_class(new_identifications_path)
-          = link_to t("nav.for_schools"), new_identifications_path, class: "govuk-header__link"
+        li.navbar-component__navigation-item--right class=active_link_class(new_publisher_session_path)
+          - if AuthenticationFallback.enabled?
```

Comment:
I reckon this should still go to `new_publisher_session_path` in both cases, no? It might be confusing to take the user straight to the login page with no explanation when the fallback is enabled.

Reply:
The fallback login page contains the explanation.